src/Compilers/Core/Portable/Symbols/Attributes/CommonPropertyWellKnownAttributeData.cs

```diff
@@ -47,5 +47,23 @@ public bool HasExcludeFromCodeCoverageAttribute
         }
 
         #endregion
+
+        #region SkipLocalsInitAttribute
+        private bool _hasSkipLocalsInitAttribute;
```

Comment:
> private bool _hasSkipLocalsInitAttribute; [](start = 7, length = 42)

If we are not going to take advantage of this attributes in VB, we should not put this field in Common layer. All fields like that should be moved to C# specific data structures.

Reply:
I'll fix that in #25673. I believe there's currently no C# specific data structure for property well known attributes. Maybe I'll have to do the same for SourceMemberMethodSymbol.